### DISCUSSION TITLE
App list and name are gone

### DISCUSSION THREAD

**Discussion by paulofpimenta** [dat-linux/community#39] [Issues]
Hi, 

I have recently updated DatLinux version and the right top panel name doesn't show Dat linux version anymore, but instead, Ubuntu version. Also, Apps list in the panel is gone.

![screen](https://github.com/dat-linux/community/assets/8817842/90081912-5b51-46f3-903e-fd4eaf78acf1)

Is this a normal behaviour ? 

Thanks for any reply

**Comment by dat-linux** [comment]
Hi, the title fix will be coming in a patch soon.

Re the missing apps, can you please output the result of 'sudo apt update' and 'cari list' commands.

thanks.

**Reply by paulofpimenta** [reply]
Hi,

Thanks for quick answer. The outputs are the following : 

```shell
Get:1 file:/var/cuda-repo-ubuntu2204-12-2-local  InRelease [1 572 B]
Get:1 file:/var/cuda-repo-ubuntu2204-12-2-local  InRelease [1 572 B]
Hit:3 https://brave-browser-apt-release.s3.brave.com stable InRelease                                                                                                                                                                      
Hit:5 http://archive.ubuntu.com/ubuntu jammy InRelease                                                                                                                                                                                     
Hit:6 https://ppa.launchpadcontent.net/alex-p/labplot/ubuntu jammy InRelease                                                                                                                                                               
Get:7 http://security.ubuntu.com/ubuntu jammy-security InRelease [110 kB]                                                                                                                                                                  
Hit:8 http://archive.ubuntu.com/ubuntu jammy-updates InRelease                                                                                                                                                                             
Hit:9 https://ppa.launchpadcontent.net/danielrichter2007/grub-customizer/ubuntu jammy InRelease                                                                                                                                            
Hit:2 https://deb.librewolf.net jammy InRelease                                                                                                                                                                                            
Hit:10 http://archive.ubuntu.com/ubuntu jammy-backports InRelease                                                                                                                                                                          
Hit:11 https://ppa.launchpadcontent.net/libreoffice/ppa/ubuntu jammy InRelease                                                                                                                                      
Hit:4 https://ubuntu.qgis.org/ubuntu jammy InRelease                                                                                                   
Hit:12 https://datlinux.gitlab.io/control-panel-1.0.1-lxqt/apt-repo stable InRelease                                             
Hit:13 https://cloud.r-project.org/bin/linux/ubuntu jammy-cran40/ InRelease                                
Hit:14 https://www.synaptics.com/sites/default/files/Ubuntu stable InRelease
Fetched 110 kB in 1s (106 kB/s)
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
All packages are up-to-date.
```

And 

```shell
Command 'cari' not found, did you mean:
  command 'cai' from deb emboss (6.6.0+dfsg-11ubuntu1)
  command 'car' from deb ucommon-utils (7.0.0-20ubuntu2)
  command 'care' from deb care (2.2.1-1build1)
  command 'card' from deb a2ps (1:4.14-7)
```

I tried to install `cari`, but it said that the package could not located. Some advice ?

Thanks again :-)

**Reply by dat-linux** [reply]
Which version of DAT Linux did you first download and install?

It looks like the cari tools are missing from your home dir. Login as a different user to confirm

**Reply by paulofpimenta** [reply]
Hi, 

The version as 1.0.1. After login with root, nothing changed. However, when i followed the command line to update from 1.0.1 (and not all users as I did before), the panel was updated I have this now :

![screen01](https://github.com/dat-linux/community/assets/8817842/5e0a242c-d996-4820-a4ae-d03ca9232fd7)

The panel name shows the datlinux version correctly, but the list of applications is still empty. Also, I am still not able to run `cari` commands. Any ideas ?

Tnks :-)

**Reply by dat-linux** [reply]
>It looks like the cari tools are missing from your home dir. Login as a different user to confirm

**Reply by paulofpimenta** [reply]
Hi,

I solved the problem. It was not related to `cari` it self, since it was present in my home folder. The issue came from the fact that I created a ~/.bash_profile to export a `PYTHON_PATH` variable.  On login, Unix systems will look for ~/.bash_profile, ~/.bash_login, and ~/.profile, in that order. Info is [here](https://www.gnu.org/savannah-checkouts/gnu/bash/manual/bash.html#Bash-Startup-Files).  Since `cari` defines and exports its environment paths from the . ~/.profile, the ~/.profile wont be read. I don't know however why this order is exclusive. Maybe Ubuntu only ? Anyways, deleting the ~/.bash_profile solved my issue.

Thank you for your help and keep the good job. Nice distro :+1:

**Reply by dat-linux** [reply]
OK, thanks for the feedback.
